src/test/java/io/reactivex/plugins/RxJavaPluginsTest.java

```diff
@@ -1890,4 +1888,106 @@ public void accept(final Throwable throwable) throws Exception {
             RxJavaPlugins.reset();
         }
     }
+
+    private static void verifyThread(Worker w, Predicate<Thread> threadPredicate) {
+        try {
+            try {
+                final AtomicReference<Thread> value = new AtomicReference<>();
```

Comment:
Then you don't even need to supply the `Thread` to the `Predicate` actually. It can verify whatever it wants.